modin/config/envvars.py

```diff
@@ -145,10 +145,20 @@ class Memory(EnvironmentVariable, type=int):
 
 class NPartitions(EnvironmentVariable, type=int):
     """
-    How many partitions to use by default
+    How many partitions to use for a Modin DataFrame (along each axis)
     """
 
     varname = "MODIN_NPARTITIONS"
+    _is_put = False
```

Comment:
```suggestion
    _is_default = True
```

Reply:
Can we add a single line comment above this too for clarifying its purpose?

Reply:
Added.